lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -690,3 +691,11 @@ def _section_metrics(course, access):
         'post_metrics_data_csv_url': reverse('post_metrics_data_csv'),
     }
     return section_data
+
+
+def is_e_commerce_course(course_key):
+    """
+    Checks if the given course is an e-commerce course or not, by checking its SKU value from
+    CourseMode records for the course
+    """
+    return len([mode.sku for mode in CourseMode.modes_for_course(course_key) if mode.sku]) > 0
```

Comment:
This function would be easier to read and easier to test if we moved all of this logic out of the return statement.  No prizes at edX for best one-liners 😉